Log/Log/LogApp.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  LogApp.swift
+//  Log
+//
+//  Created by Andras Kadar on 18.07.21.
+//
+
+import SwiftUI
+
+@main
+struct LogApp: App {
```

Comment:
lol, first I wanted to ask why do we have a log app... :D I thought it was related to the development and not the project itself.
We need to settle for a name I guess.

Reply:
Yes -.-" I think we also need to just delete this project and start a new one when we know the name